Source/WTF/wtf/OptionSet.h

```diff
@@ -125,13 +125,19 @@ template<typename E> class OptionSet {
 
         StorageType m_value;
     };
+
     using iterator = Iterator<StorageType>;
 
     static constexpr OptionSet fromRaw(StorageType rawValue)
     {
         return OptionSet(static_cast<E>(maskRawValue<E>(rawValue)), FromRawValue);
     }
 
+    static constexpr OptionSet all()
```

Comment:
This looks risky. It is setting all the bits, including the ones that don't have a valid enum value.

I could see how this function might return incorrect result as a result:
```
    constexpr bool containsAll(OptionSet optionSet) const
    {
        return (*this & optionSet) == optionSet;
    }
```

Reply:
`fromRaw` makes the input to only valid options. `return OptionSet(static_cast<E>(maskRawValue<E>(rawValue)), FromRawValue);`

Reply:
Indeed, I had missed that 👍🏻